py3status/modules/diskdata.py

```diff
@@ -170,22 +251,15 @@ def _get_free_space(self, disk):
 
         return free, used, 100 * used / total, total
 
-    def _get_diskstats(self, disk):
-        if disk and disk.startswith("/dev/"):
-            disk = disk[5:]
+    def _get_diskstats(self):
         read = 0
         write = 0
         with open("/proc/diskstats", "r") as fd:
             for line in fd:
                 data = line.split()
-                if disk:
-                    if data[2] == disk:
-                        read += int(data[5]) * self.sector_size
-                        write += int(data[9]) * self.sector_size
-                else:
-                    if data[1] == "0":
-                        read += int(data[5]) * self.sector_size
-                        write += int(data[9]) * self.sector_size
```

Comment:
Something changed between here and `master`. I think this might be it.
```
Current:
sda:   err% (    0.0 B/s)
sda:   err% (182.1 KiB/s)
sda:   err% (110.9 KiB/s)
sda:   err% (    0.0 B/s)

Master:
all:  10.9% (    0.0 B/s)
all:  10.9% (397.7 KiB/s)
all:  10.9% ( 23.8 KiB/s)
all:  10.9% (    0.0 B/s)
```

Reply:
fixed

Reply:
I find that my percents are different between both when running command `py3-cmd refresh diskdata diskdata-master` with both modules running.

Reply:
ok I'll check this one. See you in a couple hours gotta do something else :)